lib/ts/recipe/oauth2client/index.ts

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2024, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Recipe from "./recipe";
+import { RecipeInterface, APIInterface, APIOptions } from "./types";
+
+export default class Wrapper {
```

Comment:
We should expose `getAuthorisationRedirectURL`, `exchangeAuthCodeForOAuthTokens` and `getUserInfo` if practicable.
`getProviderConfig` can remain internal and the other functions can call it in the index file if necessary.

Reply:
done